model/project.go

```diff
@@ -452,10 +452,12 @@ func NewTaskIdTable(p *Project, v *version.Version, sourceRev, defID string) Tas
 
 	for _, bv := range p.BuildVariants {
 		rev := v.Revision
-		if evergreen.IsPatchRequester(v.Requester) || v.Requester == evergreen.TriggerRequester {
+		if evergreen.IsPatchRequester(v.Requester) {
 			rev = fmt.Sprintf("patch_%s_%s", v.Revision, v.Id)
 		} else if v.Requester == evergreen.TriggerRequester {
 			rev = fmt.Sprintf("%s_%s", sourceRev, defID)
```

Comment:
I don't remember if I asked this in an earlier code review or missed it. Is there any risk that this could clash with other IDs?

Reply:
The defID variable creates intentional clashes so that the same trigger and version cannot create more than 1 downstream build. Otherwise I don't expect clashes unless a generated GUID clashes